### PR TITLE
fix: peering issue between lighthouse-teku on k8s

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

File: src/cl/teku/teku_launcher.star [hunk 1]
```diff
@@ -257,7 +257,7 @@ def get_config(
         "--p2p-subscribe-all-subnets-enabled=true",
         "--p2p-peer-lower-bound={0}".format(MIN_PEERS),
         "--p2p-advertised-ip=" + PRIVATE_IP_ADDRESS_PLACEHOLDER,
-        "--p2p-discovery-site-local-addresses-enabled",
+        "--p2p-discovery-site-local-addresses-enabled=true",
```
Comment (h4ck3rk3y) [inline src/cl/teku/teku_launcher.star:260]: this is just me explicitly labelling that flag as true

File: src/cl/lighthouse/lighthouse_launcher.star [hunk 2]
```diff
@@ -188,7 +188,7 @@ def launch(
         port_id=BEACON_HTTP_PORT_ID,
         extract={
             "enr": ".data.enr",
-            "multiaddr": ".data.discovery_addresses[0]",
+            "multiaddr": ".data.p2p_addresses[0]",
```
Comment (h4ck3rk3y) [inline src/cl/lighthouse/lighthouse_launcher.star:191]: @barnabasbusa this seems like the actual fix; the discovery_address is a udp address while the p2p_address is the tcp address; changing this seems to have fixed the following configuration for me at least

```yaml
participants:
  - el_client_type: geth
    cl_client_type: lighthouse
  - el_client_type: geth
    cl_client_type: teku
  - el_client_type: geth
    cl_client_type: lighthouse
additional_services:
  - dora
```

the teku cli expects TCP in its examples; plus the other outputs of `/eth/v1/node/peers` seems to be last connecting via TCP as well

wonder why this all works on docker; maybe k8s is bottlenecked on bootnodes while docker isnt'?

**Review by barnabasbusa — APPROVED** [review]

File: src/cl/teku/teku_launcher.star (hunk 1 above)
Comment (tedim52) [reply src/cl/teku/teku_launcher.star:260]: Did this actually have any effect? If so, could be worth updating the documentation here - https://docs.teku.consensys.io/reference/cli#p2p-discovery-site-local-addresses-enabled

Comment (h4ck3rk3y) [reply src/cl/teku/teku_launcher.star:260]: this didn't, i was just confused whether its true or false so hardset it

**Pull request merged**